contributing/checklist.md

```diff
@@ -174,6 +174,21 @@ See Apple's [Accessibility Programming Guide for iOS](https://developer.apple.co
 1. Test your control on a device in VoiceOver mode and ensure the bahavior is at least as robust as UIKit.
 1. Enter YES, NO or N/A
 
+### Dynamic Type Support (If possible)
+
+
+Any component that has text, interacts with text, or lays itself out in relation to text must implement Dynamic Type.
+See Apple's [Building Apps with Dynamic Type video](https://developer.apple.com/videos/play/wwdc2017/245/) for further information.
+
+1. Set all fonts with a text style enum like `MDCFontTextStyle` or `UIFontTextStyle`, or via `UIFontMetrics`.
```

Comment:
Consider:
Set fonts with [UIFont preferredTextForStyle:]  or [UIFont mdc_perferredTextForMaterialStyle:].

Reply:
Done. I kept the font metrics thing.